packages/search/README.md

```diff
@@ -52,16 +52,14 @@ The simplest form of usage is to pass an array of strings as the `data` argument
 
 ```js
 const Component = () => {
-	const { results, query, setQuery } = useFuzzySearch( {
+	const results = useFuzzySearch( {
```

Comment:
How about we update this package in a separate PR? That's how we're usually doing here, though not mandatory and personally, I prefer it the way it's done here so it's shipped together with your changes.

Reply:
Yes I really did get carried away in this PR. This could have been separate.

> Great PR, though a bit lengthy. How do you feel about splitting it into smaller chunks?

The commits in this PR were designed to be independently reviewable. But in that case I should really just submit them as individual PRs where one depends on the other.